data/state.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
note to self: We really need to get presets in so we can get rid of this file from the repo